PokeDemo/Presenter/HomeViewPresenter.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  HomeViewPresenter.swift
+//  PokeDemo
+//
+//  Created by Rudr Bansal on 15/07/22.
+//
+
+import Foundation
+import UIKit
+
+protocol HomeViewPresenterDelegate: AnyObject {
+    func showPokemonList(pokemons: [Pokemon])
+    func showAlert(title: String, message: String)
+}
+
+typealias HomePresenterDelegate = HomeViewPresenterDelegate & UIViewController
+
+final class HomeViewPresenter {
+    
+    // MARK: Properties
+    
+    // MARK: Public
+    
+    weak var delegate: HomePresenterDelegate?
+    private let errorTitle = "Error"
```

Comment:
Move under Private MARK

Reply:
Done

Reply:
Where?

Reply:
Covered in cd97b93468f78a9f35b47f3476ee786dce2aca8a